src/expr.rs

```diff
@@ -517,6 +517,11 @@ pub(crate) fn rewrite_block_with_visitor(
     let inner_attrs = attrs.map(inner_attributes);
     let label_str = rewrite_label(label);
     visitor.visit_block(block, inner_attrs.as_ref().map(|a| &**a), has_braces);
+    let visitor_context = visitor.get_context();
+    context
+        .skipped_range
+        .borrow_mut()
+        .append(&mut visitor_context.skipped_range.borrow_mut().clone());
```

Comment:
Nit: we can consume `visitor_context` and avoid `clone`ing here, though I assume that requires an additional refactoring.

Reply:
Is it better to delete the clone than to use replace_with? The following code can also be compiled:

```
context
     .skipped_range
     .borrow_mut()
     .append(&mut visitor_context.skipped_range.borrow_mut());
```

Reply:
I think just remove clone is better than using replace_with.

Reply:
Great, this is much simpler than I first expected :smile: Thanks!